app/config.go

```diff
@@ -184,30 +186,36 @@ func InitConfig(datadir, tmNode, persistentPeers, seeds, tmRPCPort, tmPeersPort
 	if _, err := os.Stat(configFilepath); err == nil {
 		jsonFile, err = os.OpenFile(configFilepath, os.O_RDWR, os.ModePerm)
 		if err != nil {
-			panic("cannot open config json file: " + err.Error())
+			fmt.Println("cannot open config json file: " + err.Error())
+			os.Exit(1)
 		}
 		b, err := ioutil.ReadAll(jsonFile)
 		if err != nil {
-			panic("cannot read config file: " + err.Error())
+			fmt.Println("cannot read config file: " + err.Error())
+			os.Exit(1)
 		}
 		err = json.Unmarshal(b, &c)
 		if err != nil {
-			panic("cannot read config file into json: " + err.Error())
+			fmt.Println("cannot read config file into json: " + err.Error())
+			os.Exit(1)
 		}
 	} else if os.IsNotExist(err) {
 		// if does not exist create one
 		jsonFile, err = os.OpenFile(configFilepath, os.O_RDWR|os.O_CREATE, os.ModePerm)
 		if err != nil {
-			panic("canot open/create config json file: " + err.Error())
+			fmt.Println("canot open/create config json file: " + err.Error())
+			os.Exit(1)
 		}
 		b, err := json.MarshalIndent(c, "", "    ")
 		if err != nil {
-			panic("cannot marshal default config into json: " + err.Error())
+			fmt.Println("cannot marshal default config into json: " + err.Error())
+			os.Exit(1)
 		}
 		// write to the file
 		_, err = jsonFile.Write(b)
 		if err != nil {
-			panic("cannot write default config to json file: " + err.Error())
+			fmt.Println("cannot write default config to json file: " + err.Error())
+			os.Exit(1)
```

Comment:
```suggestion
			log.Fatalf("cannot write default config to json file: %s", err.Error())
```

stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work